docs/_generated_git/samples_index.rst

```diff
@@ -1,15 +0,0 @@
-  - `Arizona_consent.pdf <https://web.archive.org/web/20180210012809if_/http://www.arg.arizona.edu/papers/fmri/forms/consent.pdf>`_
```

Comment:
why this file is removed?

Reply:
oh oh sorry

Reply:
this file was autogenerated (you could see in git log).  I have reran that generation process by
```
datalad rerun 781ab17980caa062dac121d2313c8d6e0a61d9e9  # that was the last commit which generated it in master
```
and now diff is gone (you removed trailing white spaces while manually reverting it back)